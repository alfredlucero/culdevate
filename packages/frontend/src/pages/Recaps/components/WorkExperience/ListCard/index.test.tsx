import React from "react";
import { render } from "@testing-library/react";
import WorkExperienceListCard from "./index";

describe("<WorkExperienceListCard />", () => {
  test("should render without error", () => {
    const { container } = render(
      <WorkExperienceListCard onClick={() => {}} className="extra-list-class" testId="testId" />,
    );

    expect(container).toMatchSnapshot();
  });
});
