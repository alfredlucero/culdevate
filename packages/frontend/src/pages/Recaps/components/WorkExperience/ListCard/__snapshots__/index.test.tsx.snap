// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WorkExperienceListCard /> should render without error 1`] = `
<div>
  <button
    class="w-64 hover:bg-blue-100 extra-list-class"
    data-testid="testId"
    type="button"
  >
    <div
      class="bg-white shadow-md rounded px-8 py-6 h-56 flex items-center justify-center flex-col border-b-8 border-teal-400 border-solid"
      style="background-color: inherit;"
    >
      <div
        class="mb-6 flex justify-center"
      >
        <div
          class="flex justify-center items-center p-3 rounded-lg bg-gray-200 recap-icon"
        >
          <div
            class="font-sans text-3xl text-teal-400"
            id=""
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-suitcase fa-w-16 "
              data-icon="suitcase"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M128 480h256V80c0-26.5-21.5-48-48-48H176c-26.5 0-48 21.5-48 48v400zm64-384h128v32H192V96zm320 80v256c0 26.5-21.5 48-48 48h-48V128h48c26.5 0 48 21.5 48 48zM96 480H48c-26.5 0-48-21.5-48-48V176c0-26.5 21.5-48 48-48h48v352z"
                fill="currentColor"
              />
            </svg>
          </div>
        </div>
      </div>
      <h4
        class="font-mono font-bold text-gray-800 text-xl text-center"
      >
        Work Experience
      </h4>
    </div>
  </button>
</div>
`;
